diff --git a/node_modules/bullmq/dist/cjs/utils.js b/node_modules/bullmq/dist/cjs/utils.js
index b054917..0925aa9 100644
--- a/node_modules/bullmq/dist/cjs/utils.js
+++ b/node_modules/bullmq/dist/cjs/utils.js
@@ -165,11 +165,7 @@ const parseObjectValues = (obj) => {
 };
 exports.parseObjectValues = parseObjectValues;
 const errorToJSON = (value) => {
-    const error = {};
-    Object.getOwnPropertyNames(value).forEach(function (propName) {
-        error[propName] = value[propName];
-    });
-    return error;
+    return JSON.parse(JSON.stringify(value, Object.getOwnPropertyNames(value)));
 };
 exports.errorToJSON = errorToJSON;
 const INFINITY = 1 / 0;
